src/splitwisesdk/core/Expense.java

```diff
@@ -0,0 +1,24 @@
+package splitwisesdk.core;
+
+import java.util.ArrayList;
+
+public class Expense {
+    public People added_by; //doubt
+    public float total_Amount;
+    public People paid_by;  //doubt
+    public Group group;
+    public String category;
+    public ArrayList<ExpenseRatio> expenseRatios = new ArrayList<>();
+
+    public void splitEqual(){
+
```

Comment:
For every pending functions add following comment
// TODO Implement splitEqual function